spec/models/question_spec.rb

```diff
@@ -16,13 +16,64 @@
     let!(:f31) { create_question(set: set, parent: f3, name: "f31", type: "string") }
     let!(:f32) { create_question(set: set, parent: f3, name: "f32", type: "boolean") }
 
-    it 'should be set automatically' do
+    it "should be set automatically" do
       expect(f1.position).to eq 0
       expect(f2.position).to eq 1
       expect(f3.position).to eq 2
       expect(f31.position).to eq 0
       expect(f32.position).to eq 1
     end
+
+    context "with manually set position" do
+      let!(:f33) { create_question(set: set, parent: f3, name: "f33", type: "string", position: 0) }
+
+      it "goes to end anyway (position NOT respected)" do
```

Comment:
"goes to end anyway (manually set position is overridden to follow ordering rules)" would make more sense to me here. . .

Reply:
Hmm, this is what I meant: `goes to end anyway (closure_tree does not appear to respect explicit position, not sure why)`